### PR TITLE
Create freedompraise.json

### PR DESCRIPTION
**Pull request by freedompraise**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4324"></a>

### CODE REVIEW

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Hey @freedompraise I see that you have commited three other files along with your `.json` file, you need to remove the other from the commit

File: public/data/freedompraise.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name":"Praise Dike",
+  "type":"personal",
+  "bio":"Open Source Enthusiast",
+  "avatar":
```
Comment (Aadarsh805) [inline public/data/freedompraise.json:5]: you can remove this avatar field from here, and you must have links field in your profile for it to work

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]
Also, your file name is incorrect, you need to change it to `data/freedompraise.json` without the `public`

**Comment by Aadarsh805** [comment]
Feel free to ask for help, if you're stuck somewhere

**Comment by freedompraise** [comment]
> Feel free to ask for help, if you're stuck somewhere

Thank you so much. I am relatively new and trying to find my way around in this space.

**Pull request closed without merging**